xcp_d/utils/plot.py

```diff
@@ -1006,3 +1007,135 @@ def _get_tr(img):
     except AttributeError:  # Error out if not in cifti
         return img.header.get_zooms()[-1]
     raise RuntimeError("Could not extract TR - unknown data structure type")
+
+
+def plot_alff_reho_volumetric(output_path, filename, bold_file):
+    """Plot ALFF/ReHo mosaic plot for Niftis."""
```

Comment:
Can you fill out the numpydoc sections as well (Parameters, Returns, optionally Notes).